readme.txt

```diff
@@ -71,6 +71,36 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 
 == Changelog ==
 
+= 1.2.0 2020-05--8 =
+- Fix: Cast Shipping Total to float #4042 🎉barryhughes
+- Tweak: Dynamic Currency with Context API #4027
+- Fix: Remove Duplicate array entry #4049 🎉 tivnet
+- Fix: Proper display of elements in wc-admin pages when in a RTL environment. #4051
+- Enhancement: Marketing Inbox Note #4030
+- Tweak: Enable the default homepage template to be filtered #4072 🎉 stevegrunwell
+- Tweak: Create admin note if Jetpack or WooCommerce Services plugin doesn't get installed due to an error during OBW #3888
+- Fix: Update UX when knowledge base articles fail to retrieve #4133
+- Tweak: Decouple Plugins DataStore from onboarding feature #4048
+- Tweak: Move API out of Onboarding #4093
+- Dev: Add Profiler Step View Tracks #4141
+- Fix: Updated messaging after last step in OBW. #4148
+- Performance: trim down inbox note API request. #3977
+- Tweak: Update Email Marketing note. #4167
+- Fix: Reset profiler when visiting old OBW URL #4166.
+- Tweak: Adjust "demo products" verbiage to "Sample Products" #4184 🎉 jobthomas
+- Dev: Add React Testing Library #4221
+- Enhancement: New Link Component #4219
+- Performance: Use Route based code splitting to reduce bundle size #4094
+- Tweak: Use PAGE_ROOT constant to reduce redundant strings #4238 🎉 codemascot

```

Comment:
I'd be nice to get the `@` before the contributor name so they get a notification.